birdhouse/components/scheduler/pre-docker-compose-up.include

```diff
@@ -0,0 +1,3 @@
+if [ -z "${BIRDHOUSE_SCHEDULER_NOTEBOOKDEPLOY_JOB}${BIRDHOUSE_SCHEDULER_AUTODEPLOY_JOB}${BIRDHOUSE_SCHEDULER_LOGROTATE_JOB}${BIRDHOUSE_AUTODEPLOY_EXTRA_SCHEDULER_JOBS}" ] ; then
+    log WARN 'The scheduler job has been enabled but no jobs are configured to be run. Please enable some jobs or disable the scheduler'
```

Comment:
Maybe use `components/scheduler` in the message to be explicit about what it referred here, just in case there are other "scheduler" logs happening somewhere else.